cmd/registry/controller/controller.go

```diff
@@ -41,9 +55,9 @@ func ProcessManifest(
 	for _, resource := range manifest.GeneratedResources {
 		log.Debugf(ctx, "Processing entry: %v", resource)
 
-		err := ValidateResourceEntry(resource)
-		if err != nil {
-			log.FromContext(ctx).WithError(err).Debugf("Skipping resource: %q invalid resource pattern", resource)
+		errs := validateGeneratedResourceEntry(fmt.Sprintf("projects/%s/locations/global", projectID), resource)
```

Comment:
Skipping invalid entries sounds fine to me. I still think it can be valuable to pre-validate the entire manifest before processing though.

For example, we can log any issues with the manifest entries and remove them from the list of entries before processing it. That ensures the errors aren't mixed in with all the other non-error logs and can simplify the processing code.

Reply:
Makes sense, updated the code.